packages/manager/src/services/lighthouse/queries.tsx

```diff
@@ -0,0 +1,85 @@
+import { orchestratorReq } from '@app/config/orchestratorReq';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { TCreateApiKeyRes } from '../apiKeys/types';
+import { TLighthouseGenerateDTO } from './types';
+
+const lighthouseKeys = {
+  list: (webPropertyIdentifier: string, applicationIdentifier: string, env: string = '') =>
+    ['lighthouse', webPropertyIdentifier, applicationIdentifier, env] as const
+};
+
+const fetchLighthouseReportForGivenBuildId = async (
+  propertyIdentifier: string,
+  identifier: string,
+  env: string,
+  buildId: string
+): Promise<any> => {
```

Comment:
@nikhitamore  Add a todo comment to change the any to interface later

Reply:
done